root/etc/cont-init.d/60-striptracks-init

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/with-contenv bash
+
+cat <<EOF
+----------------
+>>> Striptracks Mod by TheCaptain989 <<<
+Repo: https://github.com/linuxserver/docker-mods/tree/radarr-striptracks
+
+Version: $(cat /etc/version.tc989)
+----------------
+EOF
+
+# Determine if setup is needed
+if [ ! -f /usr/bin/mkvmerge ]; then
+  echo "Running first time setup."
+
+  if [ -f /usr/bin/apt ]; then
+    # Ubuntu
+    echo "Installing MKVToolNix using apt-get"
+    apt-get update && \
+    apt-get -y install mkvtoolnix && \
+    rm -rf /var/lib/apt/lists/*
+  elif [ -f /sbin/apk ]; then
+    # Alpine
+    echo "Installing MKVToolNix using apk"
+    apk add mkvtoolnix && \
+    rm -rf /var/lib/apt/lists/*
+  else
+    # Unknown
+    echo "Unknown package manager.  Attempting to install MKVToolNix using apt-get"
+    apt-get update && \
+    apt-get -y install mkvtoolnix && \
+    rm -rf /var/lib/apt/lists/*
+  fi
+fi
+
+# Change ownership
+if [ $(stat -c '%G' /usr/local/bin/striptracks.sh) != "users" ]; then
+  echo "Changing ownership on scripts."
+  chown root:users /usr/local/bin/striptracks*.sh
```

Comment:
why not `abc:abc`, which is what all the apps run as inside the container?

Reply:
Fixed